chanfitness/chaneventstore.go

```diff
@@ -414,3 +472,33 @@ func (c *ChannelEventStore) GetUptime(channelPoint wire.OutPoint, startTime,
 		return 0, errShuttingDown
 	}
 }
+
+// recordFlapCount will record our flap count for each channel we are currently
+// tracking.
+func (c *ChannelEventStore) recordFlapCount() error {
+	log.Debugf("recording flap count for: %v channels", len(c.channels))
+
+	flapCounts := make(map[route.Vertex]map[wire.OutPoint]uint32)
+
+	for _, channel := range c.channels {
```

Comment:
is this goroutine `c.channels`' only accessor ?

Reply:
Yes, this whole subsystem is driven by the `consume` loop, so we don't need to worry about access.